code/site/libraries/plugin.php

```diff
@@ -526,7 +560,16 @@ protected function toJson()
 		}
 		else
 		{
-			return json_encode($this->get('response'));
+			require_once JPATH_SITE.'/components/com_api/views/view.json.php';

```

Comment:
later, we decided we wont need this file. Have told Umesh and Vivek but missed commenting here.